Tests/fixtures/MutationExamples/SideEffect/sampleWithImplicitReturn.swift

```diff
@@ -0,0 +1,25 @@
+func mutateMe() {
+    bar()
+}
+
+func mutateMe2() -> Void {
+    bar()
+}
+
+func mutateMe3() ->  () {
```

Comment:
If the double space after `->` is intentional, like we were discussing, probably should add a comment here so someone doesn't break it later by trying to clean it up